installer/templates/phx_single/lib/app_name_web.ex

```diff
@@ -41,9 +41,10 @@ defmodule <%= @web_namespace %> do
       use Phoenix.Controller,
         formats: [:html, :json],
         layouts: [html: <%= @web_namespace %>.Layouts]
-
-      import Plug.Conn<%= if @gettext do %>
-      import <%= @web_namespace %>.Gettext<% end %>
+      <%= if @gettext do %>
```

Comment:
This will leave trailing whitespace. I think the `<%= end %>` needs to be at the end of the previous line.